### ISSUE TITLE
Payroll: Allow all pays to be filtered by time activity type -> result in payroll picking up only time of selected activity type

### ISSUE THREAD

**Issue by dixalex** [dhanasingh/redmine_wktime#279]
Right now when you generate a payroll with multiple plays in (ex. base, overtime, weekend) -- all of them pick up ALL time entries while they should be filtered by activity type instead.

**Comment by cdhana17** [comment]
We can add a filter for activities in generate payroll page

We will put that in next release

**Comment by dixalex** [comment]
I don't understand how that will work in Generate payroll page... The goal is to have separate calculation for each rate based on the activity type. It needs to go to Payroll Settings, no?

**Comment by cdhana17** [comment]
Yes the filter will not work if you are trying to apply different rate for different activities.

How about setting up a rule like this (please see screenshot) ? Will this work ?
![Basic pay](https://user-images.githubusercontent.com/20320262/222125771-bd79e5fa-275a-4c9d-a57f-c6019996cd6c.png)

**Comment by dixalex** [comment]
Yes! That's what I was thinking. The only addition to that is it will affect the Payrol User Settings as well, because it will have to be overriden by activity type as well per user. That UI needs to be added as well.

**Comment by kikill95** [comment]
> Yes the filter will not work if you are trying to apply different rate for different activities.
> 
> How about setting up a rule like this (please see screenshot) ? Will this work ? ![Basic pay](https://user-images.githubusercontent.com/20320262/222125771-bd79e5fa-275a-4c9d-a57f-c6019996cd6c.png)

Looks good! How can I do this for myself? would be very useful for me :)

**Comment by cdhana17** [comment]
This will be available in next release

**Comment by kikill95** [comment]
> This will be available in next release

Sounds good, do you have a schedule for this? Or maybe a separate branch where I can pull this earlier?
